Modules/Core/Common/include/itkImageBase.h

```diff
@@ -810,15 +810,15 @@ class ITK_TEMPLATE_EXPORT ImageBase : public DataObject
   /** Origin, spacing, and direction in physical coordinates. This variables are
    * protected for efficiency.  They are referenced frequently by
    * inner loop calculations. */
-  SpacingType   m_Spacing;
-  PointType     m_Origin;
-  DirectionType m_Direction;
-  DirectionType m_InverseDirection;
+  SpacingType   m_Spacing{ 1.0 };
```

Comment:
Does this initialize the spacing to 1,1,1 or 1,0,0?

Reply:
@dzenanz 
> Does this initialize the spacing to 1,1,1 or 1,0,0?

1,1,1 (for a 3-D image)

Please check: `SpacingType = Vector<SpacingValueType, VImageDimension>` at https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/Common/include/itkImageBase.h#L152 so it calls the constructor `Vector(const ValueType & r)` from https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/Common/include/itkVector.h#L124